ReactViewResources/Loader.ts

```diff
@@ -120,20 +125,25 @@ export function loadPlugins(plugins: string[][], baseUrls: Dictionary<string>, m
                 // load plugin modules
                 let pluginsPromises: Promise<void>[] = [];
                 plugins.forEach(m => {
-                    const ModuleName = m[0];
-                    const NativeObjectName = m[1];
+                    const ModuleName: string = m[0];
+                    const NativeObjectName: string = m[1];
+                    const MainModule: string = m[2];
+
+                    let externalSources: string[] = m.length > 2 ? m.slice(3) : [];
```

Comment:
whats this external sources thing?

Reply:
Vendor scripts (@node_modules) or custom scripts of components created by us that are shared among views. Maybe we need a better naming here

Reply:
That was supposed to be called Plugins

Reply:
So, is this a plugin inside a plugin? 🙂 I'll update the name